Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:14.16.1-alpine
+FROM node:14.16.0-alpine
 WORKDIR /app-prototype
 COPY package.json yarn.lock ./
-RUN npx yarn@1.12.1 install
+RUN npx yarn@1.12.1 install --production
 COPY . .
 RUN npm run build
 EXPOSE 5000
```

Comment:
Not related, but shouldn't we be deleting node_modules and source code after building leaving only production files?

Reply:
First of all, I want to admit this Dockerfile was thrown together to mimic basic Heroku deploy behavior. In order for this project to deploy to Heroku as-is, with no custom build pack, there is a `Procfile` that spins up a web dyno like this:

```
web: npx --no-install serve -s
```

And the Dockerfile basically does the same thing.

This works because `serve` is a dependency in the package.json. If we were to delete node_modules, then `npx --no-install serve` would no longer work.

But there is a probably a good alternative, maybe installing `serve` globally and taking it out of package.json?

In any case I would suggest we do this in a separate PR so we can have a focused discussion about it.